src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -55,14 +57,16 @@ export default class GenomicOverview extends React.Component<IGenomicOverviewPro
         return (
             <div style={{ display:'flex' }}>
                 <Tracks mutations={this.props.mutations} key={Math.random()} sampleManager={this.props.sampleManager} width={this.props.getWidth()} cnaSegments={this.props.cnaSegments} />
-                <ThumbnailExpandVAFPlot
-                    data={this.state.frequencies}
-                    order={this.props.sampleManager.sampleIndex}
-                    colors={this.props.sampleColors}
-                    labels={labels}
-                    overlayPlacement="right"
-                    cssClass="vafPlot"
-                />
+                <If condition={this.props.mutations.length > 0}>
```

Comment:
you mean like ( condition && (<whatever/ >) ?
I mean, given the choice I prefer declarative.  We can't use it in all cases because ts doesn't recognize it as null check.  But where we can use it, I like it.  I suppose one could argue for consistency?